libevmasm/Instruction.cpp

```diff
@@ -62,6 +62,7 @@ std::map<std::string, Instruction> const dev::eth::c_instructions =
 	{ "ADDRESS", Instruction::ADDRESS },
 	{ "BALANCE", Instruction::BALANCE },
 	{ "TOKENBALANCE", Instruction::TOKENBALANCE },
+	{ "ISCONTRACR", Instruction::ISCONTRACT },
```

Comment:
Typo here, "ISCONTRACR" should be "ISCONTRACT"

Reply:
Thank you very much!